### ISSUE TITLE
Release 2.2.0

### ISSUE THREAD

**Issue by rgbkrk** [nteract/papermill#546]
Since merging in #544 I'm curious about making a release which is [likely a minor release based on the changes](https://github.com/nteract/papermill/compare/2.1.3...main). Here's a WIP change log:

* Provide help for notebooks by inspecting the `parameters` cell
* Allow `pathlib.Path`s in `execute_notebook` and `inspect_notebook`
* Powershell translation 💪

**Comment by captainsafia** [comment]
Sounds good to me! Is this something you can do?

**Comment by rgbkrk** [comment]
Yeah! I'll make the changelog now and get this shipped out.

**Comment by rgbkrk** [comment]
```
$ twine upload dist/*
Uploading distributions to https://upload.pypi.org/legacy/
Enter your username: Kyle.Kelley
Enter your password:
Uploading papermill-2.2.0-py3-none-any.whl
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 57.3k/57.3k [00:01<00:00, 43.5kB/s]
NOTE: Try --verbose to see response content.
HTTPError: 400 Client Error: File already exists. See https://pypi.org/help/#file-name-reuse for more information. for url: https://upload.pypi.org/legacy/
```

Wat. 

![image](https://user-images.githubusercontent.com/836375/98046791-ac444580-1ddf-11eb-9dda-864fc29152dd.png)

**Comment by rgbkrk** [comment]
It appears the prior 2.2.0 release was made without pushing tags or changelog @MSeal. 😬 

![image](https://user-images.githubusercontent.com/836375/98046980-07763800-1de0-11eb-89e8-80680f9d8572.png)

**Comment by rgbkrk** [comment]
Fixed up the changelog, released 2.2.1. Realizing now though that it should be 2.3.0 for the same reason I though I was bumping from 2.1.3 to 2.2.0, since the `pathlib` changes are adding functionality.
